atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/async/CassandraAsyncKeyValueServiceTests.java

```diff
@@ -113,7 +113,8 @@ private void setUpVisibleCells(Cell... cells) {
         for (Cell cell : cells) {
             when(cqlClient.executeQuery(buildGetQuerySpec(buildGetQueryParameter(cell))))
                     .thenReturn(Futures.immediateFuture(
-                            Optional.of(Value.create(RandomBytes.ofLength(10), Math.abs(new Random().nextInt())))));
+                            Optional.of(Value.create(RandomBytes.ofLength(10),
+                                    ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE)))));
```

Comment:
I don't think there's a nit bot to auto format, but spotless is unhappy:

```
* What went wrong:
Execution failed for task ':atlasdb-cassandra:spotlessJavaCheck'.
> The following files had format violations:
      atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/async/CassandraAsyncKeyValueServiceTests.java
          @@ -112,9 +112,9 @@
           ····private·void·setUpVisibleCells(Cell...·cells)·{
           ········for·(Cell·cell·:·cells)·{
           ············when(cqlClient.executeQuery(buildGetQuerySpec(buildGetQueryParameter(cell))))
          -····················.thenReturn(Futures.immediateFuture(
          -····························Optional.of(Value.create(RandomBytes.ofLength(10),
          -····································ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE)))));
          +····················.thenReturn(Futures.immediateFuture(Optional.of(Value.create(
          +····························RandomBytes.ofLength(10),
          +····························ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE)))));
           ········}
           ····}
           
  Run './gradlew :atlasdb-cassandra:spotlessApply' to fix these violations.
```

```suggestion
                    .thenReturn(Futures.immediateFuture(Optional.of(Value.create(
                            RandomBytes.ofLength(10),
                            ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE)))));
```

Reply:
ah! thanks for this, sorry I didn't see it earlier